druid/0.15/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM openjdk:8-jre
```

Comment:
Can we keep `openjdk:8-jre` ? Maintenability wise, most devs are more familiar with debian, so would prefer keeping `apt` (and maybe other possible changes).
The image size shouldn’t be an big issue, since we are using it for testing only.
Also, openjdk:8-jre is probably already pulled by some other integrations tests images or base images (e.g. elasticsearch).

Reply:
See my other answer.
Also the size issue is not because of how long it will take to pull the base image (which should be pulled only when this image is build, aka almost never), but about the size of the final image which is pulled in our testing pipeline.

Reply:
Update to alpine :)